src/package/nuspec/Microsoft.TestPlatform.nuspec

```diff
@@ -109,7 +109,6 @@
     <file src="net451\$Runtime$\Microsoft.TestPlatform.VsTestConsole.TranslationLayer.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\Microsoft.TestPlatform.VsTestConsole.TranslationLayer.dll" />
     <file src="net451\$Runtime$\Microsoft.TestPlatform.VsTestConsole.TranslationLayer.xml"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\Microsoft.TestPlatform.VsTestConsole.TranslationLayer.xml" />
     <file src="net451\$Runtime$\Microsoft.VisualStudio.ArchitectureTools.PEReader.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\Extensions\Microsoft.VisualStudio.ArchitectureTools.PEReader.dll" />
-    <file src="net451\$Runtime$\Microsoft.VisualStudio.Coverage.Analysis.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\Microsoft.VisualStudio.Coverage.Analysis.dll" />
     <file src="net451\$Runtime$\Microsoft.VisualStudio.Coverage.Interop.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\Microsoft.VisualStudio.Coverage.Interop.dll" />
```

Comment:
You can remove Microsoft.VisualStudio.Coverage.Interop.dll as well,

Reply:
Done.